app/services/proxy_test_service.rb

```diff
@@ -103,7 +103,6 @@ def api_test_path
   end
 
   def backend_api_config_path
-    return unless proxy.provider_can_use?(:api_as_product)
     proxy.backend_api_configs.first&.path
```

Comment:
We should at least ensure that it is always the same one by the order...
```suggestion
    proxy.backend_api_configs.order(:id).first&.path
```

Reply:
unless there is only 1

Reply:
my bad 😄

Reply:
#2280